src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs

```diff
@@ -140,8 +140,8 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                 }
 
                 // At this point we'll be emitting code.  Create a writer to hold it all.
-                var sw = new StringWriter();
-                IndentedTextWriter writer = new(sw);
+                using var sw = new StringWriter();
+                using IndentedTextWriter writer = new(sw);
```

Comment:
```suggestion
                using StringWriter sw = new();
                using IndentedTextWriter writer = new(sw);
```
So both have the same style?